lib/trino-parquet/src/main/java/io/trino/parquet/reader/MetadataReader.java

```diff
@@ -181,7 +181,7 @@ public static ParquetMetadata createParquetMetadata(FileMetaData fileMetaData, P
                 keyValueMetaData.put(keyValue.key, keyValue.value);
             }
         }
-        org.apache.parquet.hadoop.metadata.FileMetaData parquetFileMetadata = new org.apache.parquet.hadoop.metadata.FileMetaData(
+        io.trino.parquet.hadoop.metadata.FileMetaData parquetFileMetadata = new io.trino.parquet.hadoop.metadata.FileMetaData(
```

Comment:
After we rename the class to `FileMetadata`, we should be able to have both imported